.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
@ptitSeb Oh this has to be set to true if submitting an issue through blank template should be an option, it's up to you...

Reply:
Let's leave like that for now. Most ticket are formated well enough for now.